cmd/thanos/compact.go

```diff
@@ -172,21 +174,32 @@ func runCompact(
 		return err
 	}
 
+	confContentYamlStr, err := config.ReplaceSecret(confContentYaml)
+	if err != nil {
+		return err
+	}
+
 	bkt, err := client.NewBucket(logger, confContentYaml, reg, component.String())
 	if err != nil {
 		return err
 	}
 
-	relabelContentYaml, err := conf.selectorRelabelConf.Content()
+	reliabelContentYaml, err := conf.selectorRelabelConf.Content()
```

Comment:
```suggestion
	relabelContentYaml, err := conf.selectorRelabelConf.Content()
```
I think this change was unintentional.

Reply:
Oh I will change back.